src/pages/de/core-concepts/layouts.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Layouts
+description: Eine Einführung in Layouts - eine Astro-Komponenten-Art, mit der sich mehrere Seiten eine gemeinsame Gestaltung teilen können.
+---
+
+**Layouts** sind besondere [Astro-Komponenten](/de/core-concepts/astro-components/), die nützlich für die Erstellung wiederverwendbarer Seitenvorlagen sind.
+
+Eine Layout-Komponente wird üblicherweise verwendet, um einer [`.astro`- oder `.md`-Seite](/de/core-concepts/astro-pages/) sowohl einen **Seiten-Rahmen** (`<html>`, ` <head>` und `<body>`-Tags) als auch einen `<slot />` zur Verfügung zu stellen, der bestimmt, wo im Layout der Seiteninhalt eingefügt werden soll.
+
+Layouts enthalten häufig gemeinsame `<head>`-Elemente und gemeinsame UI-Elemente der Website, z.B. Kopfzeilen, Navigationsleisten und Fußzeilen.
+
+Layouts werden normalerweise im Verzeichnis `src/layouts` deines Projekts abgelegt.
+
+## Beispiel-Layout
+
+```astro
+---
+// Beispiel: src/layouts/MeinLayout.astro
+---
+<html>
+  <head>
+    <!-- ... -->
+  </head>
+  <body>
+    <nav>
+      <a href="#">Startseite</a>
+      <a href="#">Beiträge</a>
+      <a href="#">Kontakt</a>
+    </nav>
+    <article>
+      <slot /> <!-- Dein Inhalt wird hier eingefügt -->
+    </article>
+  </body>
+</html>
+```
+
+```astro
+---
+// Beispiel: src/pages/index.astro
+import MeinLayout from '../layouts/MeinLayout.astro';
```

Comment:
I totally understand that this is something to talk about and I really appreciate that you noticed this. Really thorough review, I like it! :)

As I commented on your other PR, it seems to be a common practice in our translations (e.g. pt-BR, which have translated tons of pages already) to localize file names and component names wherever it makes sense.

In my opinion, this highlights that these names can be freely assigned by the developer and do not have a special code meaning in Astro. I have therefore chosen to adapt this in the German translation as well.

If you feel strongly against this, we could of course decide to change it back to English, but this would also mean updating existing translations as well. I'd prefer to keep the German versions.